plugin/client_rpc.go

```diff
@@ -365,81 +366,103 @@ func init() {
 	hookNameToId["FileWillBeUploaded"] = FileWillBeUploadedId
 }
 
+type FileWillBeUploadedWrapper struct {
+	P interface{}
+}
+
+func (wrapper *FileWillBeUploadedWrapper) Read(p []byte) (int, error) {
+	return 0, fmt.Errorf("Unreachable code. Fake FileWillBeUploadedWrapper.Read called.")
+}
+
+func (wrapper *FileWillBeUploadedWrapper) Write(p []byte) (int, error) {
+	return 0, fmt.Errorf("Unreachable code. Fake FileWillBeUploadedWrapper.Write called.")
+}
+
 type Z_FileWillBeUploadedArgs struct {
```

Comment:
Is this a backwards incompatible change?

Reply:
Yes, I'll take the discussion into the main PR comment thread.